internal/server/server.go

```diff
@@ -48,10 +47,10 @@ func registerRoutes(configuration schema.Configuration, providers middlewares.Pr
 	r.GET("/api/"+apiFile, serveSwaggerAPIHandler)
 
 	for _, f := range rootFiles {
-		r.GET("/"+f, embeddedFS)
+		r.GET("/"+f, middlewares.AssetOverrideMiddleware(configuration.Server.AssetPath, embeddedFS))
```

Comment:
```suggestion
                if configuration.Server.AssetPath != "" {
		    r.GET("/"+f, middlewares.AssetOverrideMiddleware(configuration.Server.AssetPath, embeddedFS))
		} else {
		    r.GET("/"+f, embeddedFS)
		}
```

Reply:
I have the conditional check inside the middleware otherwise it'll just ignore the logic.
That way we can unconditionally apply it to all embedded filesystems and avoid several conditionals in the other area.